bower.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "simple-peer",
+  "main": "simplepeer.min.js",
+  "version": "5.11.4",
```

Comment:
Is there someway for you to use bower without a `bower.json` file in this repo? I don't want to keep the package version up-to-date across this file and `package.json`. This bower file is bound to get out of date.